test/powershell/Language/Scripting/DeserializedTypeConversion.Tests.ps1

```diff
@@ -0,0 +1,283 @@
+﻿Describe "Tests conversion of deserialized types to original type using object properties." -Tags "CI" {
+    BeforeAll {    
+        # Create new types and test functions.
+        $type1 = Add-Type -PassThru -TypeDefinition @'
```

Comment:
fixed.

---
In reply to: [78838200](https://github.com/Powershell/Powershell/pull/2118#discussion_r78838200) [](ancestors = 78838200)